integrated_channels/integrated_channel/exporters/content_metadata.py

```diff
@@ -390,10 +383,17 @@ def export(self, **kwargs):
             content_keys = self._get_catalog_content_keys(enterprise_customer_catalog)
 
             self._log_info(
-                f'Retrieved content keys: {content_keys} for past transmissions to customer: '
+                f'Retrieved {len(content_keys)} content keys for past transmissions to customer: '
                 f'{self.enterprise_customer.uuid} under catalog: {enterprise_customer_catalog.uuid}.'
             )
 
+            for content_key in content_keys:
+                self._log_info(
+                    f'Retrieved content key: {content_key} for past transmissions to customer: '
```

Comment:
i want to see / properly log each content key so i can query splunk with something like `integrated_channel_course_key=HarvardX+BOK` and see all the data.